### PR TITLE
Do not stretch NTP toggle in GUI

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Also space out the controls a bit.

Resolves: [rhbz#1972077](https://bugzilla.redhat.com/show_bug.cgi?id=1972077)

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
Here's where the differences are. Lines are hastily scribbled, so not entirely accurate.

**Before**

Screenshot_vs_main_2021-09-17_15:41:37

**After**

ntp gui cleanup

**Review by M4rtinK — APPROVED** [review]
Looks much better now, thanks! :)

File: pyanaconda/ui/gui/spokes/datetime_spoke.glade [hunk 1]
```diff
@@ -259,6 +260,8 @@
                                       <object class="GtkSwitch" id="networkTimeSwitch">
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
+                                        <property name="valign">center</property>
+                                        <property name="margin_end">6</property>
```
Comment (poncovka) [inline pyanaconda/ui/gui/spokes/datetime_spoke.glade:264]: Shouldn't we disable the expansion of the widget instead of adding the margins?

Comment (VladimirSlavik) [reply pyanaconda/ui/gui/spokes/datetime_spoke.glade:264]: The margins are in the other direction - horizontal, while the growth is vertical.

**Pull request merged**